src/pdb/cvinfo.h

```diff
@@ -19,7 +19,7 @@
 extern "C" {
 #endif
 
-#if defined(_WIN32) && defined(__cplusplus)
+#if defined(MSVC) && defined(__cplusplus)
```

Comment:
I'm not sure what you're trying to accomplish with this. The MinGW compiler should be defining `_WIN32`. This also breaks the Visual Studio build.